packages/office-ui-fabric-react/src/components/DatePicker/__snapshots__/DatePicker.test.tsx.snap

```diff
@@ -39,7 +39,7 @@ exports[`DatePicker renders default DatePicker correctly 1`] = `
           <i
             aria-hidden={true}
             className=
-                ms-DatePicker-event--without-label
+
```

Comment:
Hmm, I feel like this className should still be here but I don't see any obvious issues with the code